scribus/text/textshaper.cpp

```diff
@@ -396,8 +395,9 @@ QList<GlyphCluster> TextShaper::shape()
 			if (textRun.dir == UBIDI_RTL)
 				run.setFlag(ScLayout_RightToLeft);
 
-			lineBoundery.setPosition(firstCluster);
-			if (lineBoundery.isAtBoundary())
+			BreakIterator* lineBoundery = getLineIterator();
+			assert(lineBoundery != NULL);
```

Comment:
I don’t like this assert, we should either do nothing the rare case that and iterator can’t be created or just use some simplistic fallback e.g. the old Scribus line breaking stuff, but a hard crash is not a solution.